cluster/gce/gci/configure-helper.sh

```diff
@@ -2237,7 +2237,7 @@ function setup-fluentd {
     fluentd_gcp_configmap_name="fluentd-gcp-config-old"
   fi
   sed -i -e "s@{{ fluentd_gcp_configmap_name }}@${fluentd_gcp_configmap_name}@g" "${fluentd_gcp_yaml}"
-  fluentd_gcp_version="${FLUENTD_GCP_VERSION:-0.2-1.5.30-1-k8s}"
+  fluentd_gcp_version="${FLUENTD_GCP_VERSION:-0.3-1.5.34-1-k8s-1}"
```

Comment:
I think we have a bug in updating version. We have a huge inconsistency in labels in fluentd yaml file. Could you ensure that all labels match to the fluentd_gcp_version variable?

Reply:
Yeah, please also bump version in cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml (3 places - DS name, DS label, pod template label).
Ideally this should be also controlled via a variable, like image version, but today it's not.

Reply:
Done